### ISSUE TITLE
Better error message if `len(dv) != 3` in `Maneuver` constructor

### ISSUE THREAD

**Issue by Yash-10** [poliastro/poliastro#1270]
If I try to create a `Maneuver` object with an incorrect delta-v (i.e. doesn't consist of all three components) using:

```python
dv1 = [0, 0, 0] << u.km / u.s
dv2 = [5, 0] << u.km / u.s  # Incorrect dv

man = Maneuver((0 * u.s, dv1), (2 * u.s, dv2))
```
one gets this error:

```bash
     44         self._dts, self._dvs = self._initialize(
     45             [(_dt * u.one).value for _dt in _dts] * (_dts[0] * u.one).unit,
---> 46             [(_dv * u.one).value for _dv in _dvs] * (_dvs[0] * u.one).unit,
     47         )
     48         try:

TypeError: can't multiply sequence by non-int of type 'CompositeUnit'
```
which seems to come from `astropy` directly.

Whereas only having a single incorrect`(dt, dv)` pair:

```python
dv2 = [5, 0] << u.km / u.s  # Incorrect dv

Maneuver((2 * u.s, dv2))
```
gives a better error message:

```bash
ValueError: Delta-V must be three dimensions vectors
```

Would it be better to create a more better error message in the first case to signify that the input delta-v didn't contain all three components?

[This](https://github.com/poliastro/poliastro/blob/main/src/poliastro/maneuver.py#L42) line also suggested to change the API or error-checking, so it might be a good time to do that along with this?

Thanks!

**Comment by astrojuanlu** [comment]
Woops that's horrible. +1 on having a better error message!

**Comment by astrojuanlu** [comment]
Closed in #1271
